packaging/freebsd/openra

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+#
+# PROVIDE: openra
+# REQUIRE: LOGIN DAEMON NETWORKING mountcritlocal
+# KEYWORD: shutdown
+#
+# Add the following lines to /etc/rc.conf.local to enable the openra server:
+#
+# openra_enable="YES"
+# openra_user="<run server as this user>"
+# openra_chdir="<run server in this directory>"
+# openra_path="<path to OpenRA.Game.exe>"
+# openra_ExtraFlagslags="<set as needed>"
+#
+
+. /etc/rc.subr
+
+name=openra
+rcvar=openra_enable
+
+load_rc_config ${name}
+
+command=/usr/local/bin/screen
+pidfile=/var/run/openra.pid
+
+start_cmd="${name}_start"
+stop_cmd="${name}_stop"
+status_cmd="${name}_status"
+
+: ${openra_enable="NO"}
+: ${openra_session="openra-session"}
+: ${openra_user="openra"}
+: ${openra_chdir="/usr/home/${openra_user}"}
+: ${openra_path="/usr/home/${openra_user}/OpenRA/OpenRA.Game.exe"}
+
+: ${openra_ServerName="Dedicated_Server"}
+: ${openra_Mod="ra"}
+: ${openra_ListenPort="1234"}
+: ${openra_ExternalPort="1234"}
+: ${openra_AdvertiseOnline="False"}
```

Comment:
I'd set this to true by default. I guess not many people run a private dedicated server.